rqt_image_overlay/src/image_manager.cpp

```diff
@@ -154,7 +154,6 @@ std::optional<ImageTopic> ImageManager::getImageTopic(unsigned index)
 void ImageManager::addImageTopicExplicitly(ImageTopic imageTopic)
 {
   beginResetModel();
-  imageTopics.clear();
```

Comment:
Has this change intentionally been added to this PR? It seems like it fixes a different issue.

Reply:
Yep, this was intentional. With the test I added, I couldn't call ``addImageTopicExplicitly`` consecutively because the second call clears the topic added by the first call.

As I didn't see a point in having it there, (and so it actually "adds" the image topic, rather than clearing-then-adding).